AUTHORS.md

```diff
@@ -21,6 +21,7 @@ The following wonderful people contributed directly or indirectly to this projec
 - Nikolai Schwertner <https://github.com/nschwertner>
 - Pascal Pfiffner <https://github.com/p2>
 - Raheel Sayeed <https://github.com/raheelsayeed>
+- Svetlana Grebennikova <https://github.com/LanaNYC>
```

Comment:
Yay! Thanks for your work on that PR.

You could have done this in the original PR too. I'll approve this but wait to merge it until the actual PR lands.